lib/awful/widget/watch.lua

```diff
@@ -0,0 +1,84 @@
+---------------------------------------------------------------------------
+--- Watch widget.
```

Comment:
I don't mind this screenshot having another color theme. Feel free to leave it like this.